### DISCUSSION TITLE
Markdown

### DISCUSSION THREAD

**Discussion by Fresh-Teacher-Uganda** [community/community#19918] [Discussions] [answered]
What is the markdown code for lists?

**Comment by Fresh-Teacher** [comment, marked as answer]
Markup : * Bullet list
              * Nested bullet
                  * Sub-nested bullet etc
          * Bullet list item 2

-OR-

 Markup : - Bullet list
              - Nested bullet
                  - Sub-nested bullet etc
          - Bullet list item 2 
          
 Markup : 1. A numbered list
              1. A nested numbered list
              2. Which is numbered
          2. Which is numbered
          
 Markup : - [ ] An uncompleted task
          - [x] A completed task
